build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
```

Comment:
for spock groovy pluging should be added

Reply:
done